docs/governance/README.md

```diff
@@ -0,0 +1,116 @@
+# .NET Standard governance
+
+The .NET Standard represents the set of APIs that all .NET implementations have
+to provide. Over time, this list has to grow but it's critical that the set
+remains implementable by all current and future .NET implementations.
+
+To ensure that's the case, there is a formal process for proposing and accepting
+APIs into the .NET Standard. This document outlines the process.
+
+## Review board
+
+Today, we have several different .NET implementations:
+
+* .NET Core
+* .NET Framework
+* Mono
+* Unity
+* Xamarin
+
+Some of them have drastically different constraints, based on their runtime
+environment (for instance, Xamarin for iOS cannot perform runtime code
+generation, based on Apple's app store policy).
+
+To ensure we don't end up adding large chunks of API surface that cannot be
+implemented, we have a review board that has to sign-off on API additions to
+the .NET Standard. The board is comprised of representatives from the following
+groups:
+
+  * **.NET platform**. The rationale here is that most, if not all, of the APIs
+    that are part of .NET Standard are implemented and evolved by the .NET
+    platform team.
+  * **Xamarin & Mono**. While Mono mostly copy code from .NET Framework & .NET
+    Core base class library, changes and extensions can impact their ability to
+    support the .NET Standard. Thus, we need to coordinate any changes with
+    Mono.
+  * **Unity**. Same rationale as for Xamarin & Mono.
+
+The chairman of the review board is [@migueldeicaza](https://github.com/migueldeicaza).
+For the most part, we strive to make decisions based on consensus, but as it is
+with every group of engineers, consensus can be impossible to achieve at times
+so we need to have an ultimate tie breaker. And Miguel has a lot of expertise
+and experience building .NET implementations that are supported by multiple
+parties.
+
+Please note that the member list isn't meant to be closed: as more platform
+vendors and API drivers appear, the review board will expand accordingly.
+
+## Process
+
+* **Anybody can submit proposals** for API additions to the .NET Standard.
+  Those will be tracked as issues and labeled with the [netstandard-api] label.
+* **Acceptance requires**
```

Comment:
You didn't list the API review board signoff. I assumed the .NET Standard request would be downstream of approving the final API shape. Running those processes potentially in parallel could make things quite complicated especially if the review board suggests changes.

I am not sure how much precedent we have for API review of API's whose initial implementation is not in .NET Framework or .NET Core. That seems like something we can fix, though.

Reply:
Particularly since you suggest that additions must be mature API's. That seems to imply that they've been okayed by API review.

Reply:
Good point. I've changed the wording above to demand stable APIs as input, as opposed to net-new or prototypes. I don't want the review board to become another API design review -- the intend is that the board only has to decide whether the API should be in the standard.

Reply:
You've mentioned that some future API's could appear first in Mono, as opposed to appearing first in .NET Core which seems to almost invariably be the case right now (I think). Is the API review board set up for that? ie., do we have precedent for Xamarin folk presenting API to it, and being fully part of decision making there?

Reply:
Also (and I expect this would be super rare), what happens if different platforms independently add conflicting APIs?